examples/awaits.rb

```diff
@@ -15,21 +19,25 @@
   # Pick a number between 1 and 10
   magic = rand(1..10)
 
+  # Tell the user that we're ready.
+  event.respond "Can you guess my secret number? It's between 1 and 10!"
+
   # Await a MessageEvent specifically from the invoking user.
+  # Timeout defines how long a user can spend playing one game.
+  # This does not affect subsequent games.
   #
-  # Note that since the identifier I'm using here is `:guess`,
-  # only one person can be playing at one time. You can otherwise
-  # interpolate something into a symbol to have multiple awaits
-  # for this "command" available at the same time.
-  event.user.await(:guess) do |guess_event|
+  # You can omit the options hash if you don't want a timeout.
+  event.user.await!(timeout: 300) do |guess_event|
     # Their message is a string - cast it to an integer
     guess = guess_event.message.content.to_i
 
-    # If the block returns anything that *isn't* `false`, then the
+    # If the block returns false, then the
     # event handler will persist and continue to handle messages.
+    # If it returns true, it will cancel itself.
```

Comment:
```suggestion
    # If the block returns anything that *isn't* true, then the
    # event handler will persist and continue to handle messages.
```

Reply:
Had to improvise since the old comment from discordrb/discordrb:master was outdated. Can go with this one.